packages/sdk/src/types/PageTypes.ts

```diff
@@ -13,3 +13,7 @@ export type PageSize = {
     width: number;
     height: number;
 };
+
+export interface SnapshotSettings {
+    largestAxisSize?: number | null;
```

Comment:
```suggestion
    maxDimensionSize?: number | null;
```

Reply:
it has been merged on the engine side, so this change would require to make changes on the engine side as well

Reply:
so this means you'll do it or this means it's not possible? I really think it's a non descriptive vague name

Reply:
I would leave it as it is for now, considering the current code

Reply:
So what are we going to do now? We can always add a comment explaining what it means.

cc @brapoprod @evan-mcgeek

Reply:
I guess that's the least we should do idd